src/views/PackageDetail.vue

```diff
@@ -112,7 +112,7 @@
               </nav>
               <nav class="panel">
                 <p class="panel-heading">
-                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'box-open']" />Package info
+                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'box-open']" />Module Info
```

Comment:
```suggestion
                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'box-open']" />Module info
```

Because the rest of the titles on the panels aren't capitalized after the first word, I think "info" should be lowercase as well. I noticed #266 a few minutes ago though, so perhaps we can discuss this further in that issue (after this PR is merged).

Reply:
ok - absolutely